<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WSchedule</class>
 <widget class="QWidget" name="WSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="searchLineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="sortButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
